ansible/roles/base_setup/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Create storage directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: '0755'
+  with_items:
+    - /store/keep
+    - /store/tmp
+
+- name: Mount volumes
+  mount:
+    path: "{{ item.path }}"
+    src: "{{ item.device }}"
+    fstype: ext4
+    state: mounted
+  with_items:
+    - { path: "/store/keep", device: "/dev/nvme1n1" }
```

Comment:
I'm guessing NADCloud knows to name the devices this? I'm a little confused about where their handoff to us is.